Updates/1559_Saltgurka-Molten_Core_Fixes.sql

```diff
@@ -0,0 +1,454 @@
+
+-- Firesworn follows Garr. To be blizzlike they should remain in a perfect circle around him at all times, but that's not possible with the current linking system
+UPDATE `creature_linking_template` SET `flag`=1543 WHERE `entry`=12099 AND `master_entry`=12057; -- Old flag: 7
+
+-- ************************
+-- Core Hounds
+-- ************************
+-- Pathing for Ancient Core Hound Entry: 11673 'TDB FORMAT' 
+-- Original had duplicate points
+SET @GUID := 56853;
+UPDATE `creature` SET `spawndist`=0,`MovementType`=2,`position_x`=1039.667,`position_y`=-791.0968,`position_z`=-151.0096 WHERE `guid`=@GUID;
+DELETE FROM `creature_movement` WHERE `id`=@GUID;
+INSERT INTO `creature_movement` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`) VALUES
+(@GUID,1,1039.667,-791.0968,-151.0096,0),
+(@GUID,2,1026.315,-767.8003,-156.6493,0),
+(@GUID,3,1038.774,-761.6977,-152.658,0),
+(@GUID,4,1064.627,-748.9409,-151.7516,0),
+(@GUID,5,1038.774,-761.6977,-152.658,0),
+(@GUID,6,1024.332,-768.7155,-156.9002,0),
+(@GUID,7,1039.641,-791.0518,-150.9892,0),
+(@GUID,8,1051.314,-799.6896,-151.9713,0),
+(@GUID,9,1040.764,-815.1115,-152.1656,0),
+(@GUID,10,1051.314,-799.6896,-151.9713,0);
+
+-- Pathing for Ancient Core Hound Entry: 11673 'TDB FORMAT' 
+-- Original one had some duplicate points
+SET @GUID := 56856;
+UPDATE `creature` SET `spawndist`=0,`MovementType`=2,`position_x`=706.5988,`position_y`=-608.756,`position_z`=-209.7862 WHERE `guid`=@GUID;
+DELETE FROM `creature_movement` WHERE `id`=@GUID;
+INSERT INTO `creature_movement` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`) VALUES
+(@GUID,1,706.5988,-608.756,-209.7862,0),
+(@GUID,2,694.3823,-643.1285,-209.7859,0),
+(@GUID,3,691.9941,-667.7454,-209.6549,0),
+(@GUID,4,712.6238,-699.4688,-209.7263,0),
+(@GUID,5,696.8894,-727.3798,-209.4279,0),
+(@GUID,6,672.7046,-770.8894,-209.0702,0),
+(@GUID,7,696.8894,-727.3798,-209.4279,0),
+(@GUID,8,712.6238,-699.4688,-209.7263,0),
+(@GUID,9,691.9941,-667.7454,-209.6549,0);
+
+-- Pathing for Ancient Core Hound Entry: 11673 'TDB FORMAT' 
+-- At the second resting area it should be moving around "randomly" on the spot. It didn't on the old path.
+SET @GUID := 56851;
+UPDATE `creature` SET `spawndist`=0,`MovementType`=2,`position_x`=1144.05,`position_y`=-662.6844,`position_z`=-131.4626 WHERE `guid`=@GUID;
+DELETE FROM `creature_movement` WHERE `id`=@GUID;
+INSERT INTO `creature_movement` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`waittime`) VALUES
+(@GUID,1,1144.05,-662.6844,-131.46260,0,0),
+(@GUID,2,1160.286,-696.8795,-132.48630,0,0),
+(@GUID,3,1129.031,-721.049,-139.58040,0,0),
+(@GUID,4,1118.512,-712.1383,-144.25850,0,0),
+(@GUID,5,1104.687,-722.1303,-146.98320,0,0),
+(@GUID,6,1090.824,-731.3634,-149.98680,0,0),
+(@GUID,7,1070.791,-717.5623,-153.48310,0,0),
+(@GUID,8,1080.754,-681.5721,-159.87260,0,0),
+(@GUID,9,1081.626,-681.3043,-159.6755,100,12000),

```

Comment:
Yeah we had that conversation a while ago at https://github.com/classicdb/database/issues/504
It is still open because of this very dog.

I'd ask Cyberium if he could have a look at it, but there is way too much work currently for him, this has to wait. It is used almost everywhere, so more or less important.

Just leave it like it is then, I won't forget about this dog, I'll change that when it's possible.